app-testing/src/pages/device_landing.py

```diff
@@ -88,6 +197,119 @@ def get_recent_protocol_runs_header_text(self) -> str:
             return ""
         return element.text
 
+    def get_mag_deck_image(self) -> Optional[WebElement]:
+        """Get the mag deck image on device detail page."""
+        image: Element = Element(
+            (By.XPATH, f"//img[@alt='magneticModuleV1']"),
+            f"the mag deck image on device detail page.",
+        )
+        return self.base.clickable_wrapper(image, 5)
+
+    def get_thermocycler_deck_image(self) -> Optional[WebElement]:
+        """Get the thermocycler deck image on device detail page."""
+        image: Element = Element(
+            (By.XPATH, f"//img[@alt='thermocyclerModuleV1']"),
+            f"the thermocycler deck image on device detail page.",
+        )
+        return self.base.clickable_wrapper(image, 5)
+
+    def get_tem_deck_image(self) -> Optional[WebElement]:
+        """Get the temp deck image on device detail page."""
+        image: Element = Element(
+            (By.XPATH, f"//img[@alt='temperatureModuleV1']"),
+            f"the temp deck image on device detail page.",
+        )
+        return self.base.clickable_wrapper(image, 5)
+
+    def get_left_mount_pipette_device_detail(self) -> Optional[WebElement]:
+        """Get the left mount pipette on device detail page."""
+        text: Element = Element(
+            (
+                By.XPATH,
+                f"//div[@data-testid='PipetteCard_display_name_P10 Single-Channel GEN1']",
+            ),
+            f"the left mount pipette on device detail page.",
+        )
+        return self.base.clickable_wrapper(text, 5)
+
+    def get_protocol_name_device_detail_slideout(self) -> Optional[WebElement]:
+        """Get the protocol name on device detail page slide out."""
+        text: Element = Element(
+            (
+                By.XPATH,
+                f"//div[@data-testid='Slideout_body_Choose protocol to Run on opentrons-dev']//div",
+            ),
+            f"the protocol name on device detail page slide out.",
+        )
+        return self.base.clickable_wrapper(text, 5)
+
+    def get_right_mount_pipette_device_detail(self) -> Optional[WebElement]:
+        """Get the right mount pipette on device detail page."""
+        text: Element = Element(
+            (
+                By.XPATH,
+                f"//div[@data-testid='PipetteCard_display_name_P300 Single-Channel GEN1']",
+            ),
+            f"the right mount pipette on device detail page.",
+        )
+        return self.base.clickable_wrapper(text, 5)
+
```

Comment:
This does not have to change but if we had a dev server or robot with a different pipette mounted on the right we could make this function handle that.

call it like:
`get_right_mount_pipette_device_detail(pipette_id="P300 Single-Channel GEN1")`

```suggestion
    def get_right_mount_pipette_device_detail(self, pipettte_id: str) -> Optional[WebElement]:
        """Get the right mount pipette on device detail page."""
        text: Element = Element(
            (
                By.XPATH,
                f"//div[@data-testid='PipetteCard_display_name_{pipettte_id}']",
            ),
            f"The right mount pipette ({pipette_id}) on device detail page.",
        )
        return self.base.clickable_wrapper(text, 5)
```

Reply:
We are saying it is on the right but we don't do any check that it is on the right.

Reply:
Done